### ISSUE TITLE
Wrong picture content

### ISSUE THREAD

**Issue by PaulElHilany** [minutelabsio/Tree-of-Life-Explorer#53]
Dear minutelabsio team,

the tag Puma (genus) has pictures of: A football player, an attack helicopter, a band, and something like a S.W.A.T. team.

Best,
Paul.

**Comment by wellcaffeinated** [comment]
Yes... that's part of the trouble with pulling images from a simple wiki commons API unfortunately. Hopefully we'll find a better way to do that eventually

**Comment by PaulElHilany** [comment]
Okay! It didn't bother me though; it was just to make sure that you guys are aware of that.
